src/campaigns/ai/chat/aiChat.types.ts

```diff
@@ -0,0 +1,19 @@
+export enum AiChatFeedbackType {
+  positive = 'positive',
+  negative = 'negative',
+}
+
+export type AiChatMessage = {
+  role: 'user' | 'system' | 'assistant'
+  content: string
+  createdAt?: number
+  // messageId?: string
+  id?: string
+  usage?: number
+}
+
+export type AiChatData = {
+  messages: AiChatMessage[]
+  // TODO: should feedback be on the individual message instead of on the AIChat object?
```

Comment:
I'm thinking, when this is addressed, `AiChatFeedback` should just be an enumeration of `possitive`, `negative` or `null` on each message record.  No need to persist the negative feedback text, since that's only ever used to send the slack message, it's never read in the UI.